src/todo/UI.lua

```diff
@@ -101,13 +101,27 @@ function todo.create_task_table(frame, player)
         caption = ""
     })
 
+    table.add({
+        type = "label",
+        style = "todo_label_default",
+        name = "todo_title_top",
+        caption = ""
+    })
+
     table.add({
         type = "label",
         style = "todo_label_default",
         name = "todo_title_down",
         caption = ""
     })
 
+    table.add({
+        type = "label",
+        style = "todo_label_default",
+        name = "todo_title_bottom",
+        caption = ""
+    })
+
```

Comment:
If I read this clearly the order of the buttons is :
[UP] [TOP] [DOWN] [BOTTOM]
I think it would be better to have :
[TOP] [UP] [DOWN] [BOTTOM]